### DISCUSSION TITLE
ESMFold with "fold switching" proteins

### DISCUSSION THREAD

**Discussion by ga01** [facebookresearch/esm#345] [ESM Metagenomic Atlas]
Great tool;
 
I wonder whether "fold switching" proteins could be (or even were?) used to improve the predictions of ESMFold. 

For example, the proteins 
TTYKLILNLKQAKEEAIKELVDAGTAEKYFKLIANAKTVEGVWT**L**KDEIKTFTVTE
and
TTYKLILNLKQAKEEAIKELVDAGTAEKYFKLIANAKTVEGVWT**Y**KDEIKTFTVTE

differ only by one (!) amino acid, but their folds are completely different (3a vs. 4b+a).
(see the PNAS paper by Alexander et al. 2009 for details, https://www.pnas.org/doi/10.1073/pnas.0906408106 )

A quick check of ESMFold predictions indicates that their predicted structures are quite uncertain, nevertheless b-sheets do appear in the prediction of the second sequence, so it might not be hopeless. I wonder whether comparable cases were tested on a larger scale, or used in the development of the tool?

Cheers, 
George

**Comment by tomsercu** [comment]
This is very interesting, thanks for sharing these examples!
Here it is using the atlas fold UI: [foldswitching 1](https://esmatlas.com/resources/fold/result?fasta_header=%3Efoldswitching%201&sequence=TTYKLILNLKQAKEEAIKELVDAGTAEKYFKLIANAKTVEGVWTLKDEIKTFTVTE), [foldswitching 2](https://esmatlas.com/resources/fold/result?fasta_header=%3Efoldswitching%202&sequence=TTYKLILNLKQAKEEAIKELVDAGTAEKYFKLIANAKTVEGVWTYKDEIKTFTVTE).
Model predictions are low confidence (mostly for the second), but this definitely isn't a complete failure.

We studied some case studies internally, but not publishing a systematic study on this.

**Comment by ga01** [comment]
Yes, their predicted structures are very encouraging, ESMFold definitely captures something relevant about the fold-switching. One could compile a set of structure pairs with 90-95% sequence identity, but a much lower TM-score than the average in the PDB (excluding domain-swapped stuff); maybe it would be useful in benchmarking and improving the accuracy of future versions of ESMfold (especially in the prediction of the effects of mutations).

But that would also touch on another problem, the estimation of conformational ensembles, which I guess is one of the next steps.

George
